Pages/Search.cshtml

```diff
@@ -41,10 +43,17 @@
                             </div>
                         </div>
                         <div class="text-center">
-                            <button type="submit" class="btn btn-primary">Search</button>
+                            <button type="submit" class="btn btn-primary btn-lg">Search Breweries</button>
                         </div>
+                      
                     </form>
-
+                  
+                    @if (string.IsNullOrEmpty(Model.State) && string.IsNullOrEmpty(Model.BreweryType))
+                    {
+                        <div class="alert alert-warning mt-3">
+                            Please enter at least one search criterion.
+                        </div>
```

Comment:
The rationale for the conditional check and alert message is to ensure that users are prompted to provide at least one search criterion before submitting the form. By checking if both the State and BreweryType fields are empty or null, it prevents the submission of an incomplete search. If both fields are empty, an alert message is displayed, guiding the user to enter a valid search criterion. This enhances the user experience by offering clear feedback and preventing unnecessary searches. The alert is styled using Bootstrap's alert and alert-warning classes for visibility and emphasis, making it clear to the user what action is required.

Reply:
Good point.
Alternatively, can you default to showing all breweries if no search term is entered?